public/js/util/utils.js

```diff
@@ -97,6 +97,12 @@ function entries(obj) {
   return Object.keys(obj).map(k => [k, obj[k]]);
 }
 
+function mapObject(obj, iteratee) {
```

Comment:
i initially wrote this for serializing the store and i don't have the heart to kill it. I've wanted other times to